src/transformers/__init__.py

```diff
@@ -1271,6 +1281,9 @@
         name for name in dir(dummy_flax_objects) if not name.startswith("_")
     ]
 
+# Vision-backed objects
+if is_vision_available():
+    _import_structure["models.vit"].append("ViTFeatureExtractor")
```

Comment:
Can we put it in the same test as the `ImageFeatureExtractionMixin`?

Reply:
What do you mean by "putting in the same test"?

Reply:
Not writing a new `if is_vision_available()` test since there is already one before the modeling imports (line 347 on master)